keps/sig-storage/596-csi-inline-volumes/README.md

```diff
@@ -177,34 +235,13 @@ also set an entry with this key in the `NodePublishRequest.volume_context`:
 * `csi.storage.k8s.io/ephemeral`: `true` for ephemeral inline volumes, `false` otherwise
 
 ### Secret reference
+
 The secret reference declared in an ephemeral inline volume can only be used with namespaces from pods where it is referenced.  The `NodePublishSecretRef` is stored in a `LocalObjectReference` value:
 * `LocalObjectReference` do not include a namespace reference.  This is to prevent reference to arbitrary namespace values.
 * The namespace needed will be extracted from the the pod spec by the Kubelet code during mount.
 
-### Specifying allowed inline drivers with `PodSecurityPolicy`
```

Comment:
nit: duplicate `the` in `...... from the the pod spec ......` on line 241.

Reply:
Updated, thanks